app/src/main/kotlin/Reversi/ReversiState.kt

```diff
@@ -17,11 +17,6 @@ class ReversiState : Cloneable {
         squares[size/2 - 1][size/2] = ReversiSquare.DARK
         squares[size/2][size/2 - 1] = ReversiSquare.DARK
 
```

Comment:
What was this FEASIBLE attribute for in the first place?

Reply:
It used to track which moves were possible for the next player. I thought this could help improve performance at one point. However, given how much much the board changes after each move, this list get outdated really quickly so it was more hinderance than help so I removed it.